### ISSUE TITLE
Wrong recordings shown in Conflicting recordings window

### ISSUE THREAD

**Issue by kmdewaal** [MythTV/mythtv#348]
* **Platform**: Fedora 33

* **MythTV version**: master

* **Package version**: build from source

* **Component**:

### What steps will reproduce the bug?

Given a system with two video sources, a DVB-C source Ziggo and a DVB-S/S2 satellite source Freesat.
There are two programs scheduled to record on Ziggo.
There is one program scheduled to record on Freesat.
All scheduled programs show up green in the Program Guide.

Then an attempt is made to schedule a second, overlapping, program recording on Freesat. This fails because there is only one tuner and the second program is on a different multiplex. This appears in the program guide as a conflicting recording.
See screenshot below, Record_Conflicting_2021-03-29 20-26-54.png.
![Record_Conflicting_2021-03-29 20-26-54](https://user-images.githubusercontent.com/10491032/112883695-eab00900-90ce-11eb-8541-4039523d3c41.png)

When pressing Enter or Space a window is shown that explains that other programs with a higher priority will be recorded instead, followed by a list of the programs that will be recorded.
See screenshot below, Conflicting_other_recordings_2021-03-29 20-24-58.png.
![Conflicting_other_recordings_2021-03-29 20-24-58](https://user-images.githubusercontent.com/10491032/112884032-4b3f4600-90cf-11eb-9cc8-8dfb8a248b8e.png)

The list of programs that will be recorded is wrong for two reasons:
1. The programs shown will actually be recorded but they are from video source Ziggo.
2. The program that will be recorded from the Freesat video source is NOT listed.

The correct behavior is:
1. Show only the programs that will be recorded from the Freesat video source.

To summarize, the "Conflicting recordings" window should only show the recordings that will be recorded from the same video source and ignore the recordings scheduled on other video sources.

### Additional information

If my recollection is correct the list of conflicting recordings has been completely missing for a while until it has been restored in commit 93c278d430c9f7252aff762783556568433cda02

**Comment by kmdewaal** [comment]
Fixed in commit  70d90a68de1c3b9011ea26b7f0ddc630604ca77d.
